packages/react-core/src/default-error-component.tsx

```diff
@@ -33,7 +33,7 @@ const DefaultErrorComponent = ({
     <div>
       <h2>Error loading component</h2>
       {typeof error === 'string' && <p>{error}</p>}
-      {error?.cause && <p>{error.cause}</p>}
+      {error?.cause && typeof error?.cause !== 'object' && <p>{error.cause}</p>}
```

Comment:
@Hyperkid123 NIT and not blocking, but should we make this even stricter and check `typeof error?.cause === 'string'` instead? I'm not sure this would make sense to render a boolean, number, etc...

Reply:
I also see we're creating the ErrorWithCause type above in this file. We may eventually want to converge on one type/class between the dynamic plugin SDK and this logic: https://github.com/openshift/dynamic-plugin-sdk/blob/main/packages/common/src/errors/ErrorWithCause.ts (though this object is mainly used for error handling in the SDK).